cli/src/main.cpp

```diff
@@ -11,6 +13,9 @@
 	#include <vendor/qcommandlineparser.h>
 #endif
 
+void loadMoreDetails(const QList<QSharedPointer<Image>> &images);
+QJsonObject serializeImg(const Image *image, const QMap<QString, Token>& tokens);
+void writeToFile(QString filename, QByteArray data);
```

Comment:
This function does not seem to be used, or did I miss something?

Reply:
Currently not, but it was/will be usefull for debugging the json output.